app/src/main/java/org/eyeseetea/malariacare/network/ServerAPIController.java

```diff
@@ -191,9 +189,9 @@ public static String getProgramUID() {
     /**
      * Returns hardcoded credentials for its use in sdk
      */
-    public static UserCredentials getSDKCredentials() {
+ /*   public static UserCredentials getSDKCredentials() {
         return SdkLoginController.getCredentials(getUserPush(), getPassPush());
```

Comment:
@xurxodev should be this commented? this method is necessary to do push in lao and myanmar
maybe can be changed to:

        return new UserCredentials(getUserPush(), getPassPush());